problems/removeEvenStrings.js

```diff
@@ -6,8 +6,7 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+    return arr.filter(onlyEven =>{ return onlyEven.length % 2 === 1})
```

Comment:
If the anonymous function is written in one line, then there is no need for the curly brackets after the fat arrow and no need for the return statement